pyvera/__init__.py

```diff
@@ -293,13 +307,7 @@ def set_value(self, name, value):
                         'action': 'Set' + name,
                         'newLoadlevelTarget': value}
                 else:
-                    payload = {
```

Comment:
I would have moved the LoadLevelTarget logic to a decedent class method override of get_payload but I don't know which entity that should be a part of. If you know which entity needs the parameter constructed that way I'd be happy to move it to the get_payload method of that entity.

The reason I moved it to a method is because Thermostat required a different dictionary than either of the original dictionaries provided (the name=LoadLevelTarget or Else case prior to my change).

I'll clean this up a bit so we can re-use creating the majority of the dictionary, while only adding the last parameter to the dictionary but allowing the sub class or base class to create that parameter.

After my change should work as follows:
1. create dictionary with id, output_format, DeviceNum, serviceId and action populated.
2. add last parameter to dictionary
    2a. (if themostat) New<name> = x (provided by VeraThermostat class)
    2b. (if name != LoadLevelTarget) new<name>Value = x (provided by base class)
    2c. (if name == LoadLevelTarget) newLoadLevelTarget = x (provided by if statement, unless you can tell me which class needs this)

Make sense?